### ISSUE TITLE
Access to latest Vocabulary Mappings implementation

### ISSUE THREAD

**Issue by eltonfss** [LiUSemWeb/HeFQUIN#318]
Hi, 

I'm currently investigating state-of-the-art solutions that are capable of translating SPARQL queries in a global schema to SPARQL queries in local schemas via vocabulary mappings, and I came across one of the latest publications related to this repository: [Considering Vocabulary Mappings in Query Plans for Federations of RDF Data Sources](https://link.springer.com/chapter/10.1007/978-3-031-46846-9_2).

The publication refers to [HeFQUIN](https://github.com/LiUSemWeb/HeFQUIN) and to [HeFQUIN-VocabMappingsExperiments](https://github.com/LiUSemWeb/HeFQUIN-VocabMappingsExperiments), but the second link leads me to a 404 page from GitHub. 

Would you be able to give me access to it or make it public so I can test your implementation and see if I could use and/or extend it?
If you could also share any further documentation or examples you might have of vocabulary mappings and queries related to them it would be very helpful as well.

Best Regards

**Comment by chengsijin0817** [comment]
Hi Elton, thanks a lot for your interest. 
I've made the [HeFQUIN-VocabMappingsExperiments](https://github.com/LiUSemWeb/HeFQUIN-VocabMappingsExperiments/tree/main) repository public. If you have any questions or if there's anything unclear, please feel free to reach out.

**Comment by hartig** [comment]
Also, Elton, please let us know of the findings of your investigation. We are eager to read about it!

**Comment by eltonfss** [comment]
Thank you @chengsijin0817 and @hartig! So far this is the most recent and accessible solution I've been able to find. 

Another interesting solution was [SPARQL-RW](https://www.w3.org/2001/sw/wiki/SPARQL-RW), but its latest publication was about 10 years ago and I could not yet find any publicly available source code of the solution for testing.

Another one I found that was a bit more recent was [sparql-translator-complex-alignment](https://framagit.org/IRIT_UT2J/sparql-translator-complex-alignment) whose [publication](https://hal.science/hal-02102384/document) dates from 2016.

Finally, I also looked into [SPARQLoid](https://dl.acm.org/doi/abs/10.5555/2887379.2887385) which was published in ISWC 2012, but I could not find the source code for testing it anywere as well.
